packages/core/src/connectors/aliyun-dm/index.ts

```diff
@@ -38,12 +38,20 @@ export interface AliyunDmConfig {
   templates: Template[];
 }
 
+const testMessageTemplate: Template = {
+  type: 'Test',
+  subject: 'It works',
+  content: 'Congratulations!',
+};
+
 export const sendMessage: EmailSendMessageFunction = async (address, type, data) => {
   const config: AliyunDmConfig = await getConnectorConfig<AliyunDmConfig>(
     metadata.id,
     metadata.type
   );
-  const template = config.templates.find((template) => template.type === type);
+  const template = [...config.templates, testMessageTemplate].find(
+    (template) => template.type === type
+  );
```

Comment:
do we need to hard code these or we just insert a test template when seeding?

Reply:
Hard code would be better:
1. user (developer) don't need to see this in config
2. prevent deleting by accident.

Reply:
some developers will find it annoying since the test template may not fulfill connector's requirement or they do not wanna see it. plus:
1. we'll also have a double-check process in AC when deleting anything.
2. user can choose to "generate a template" when they are trying to enable a connector in AC.
@logto-io/eng any ideas?

Reply:
Make sense, another point is that the test message is not important, so is all right if some developers delete them manully.
I'll remove this code and mention this in the document.

Reply:
both proposals are good.

Reply:
cool

Reply:
seed: LOG-1389